### DISCUSSION TITLE
Migrate from Google App Script IDE To App Script Starter

### DISCUSSION THREAD

**Discussion by coreyjansen** [labnol/apps-script-starter#59] [General]
Since the code in App Scripter IDE and the code in the App Script Starter is written very differently using imports, exports... what is the best way to go from a current script that is written in the Google App Script IDE and clone it with clasp so you can use with the starter kit?

**Comment by coreyjansen** [comment]
And to be more clear i mean going from Google App Script IDE that doesn't use modules to using app script starter that does use modules

**Comment by Audiobrad** [comment]
Looking for the same
